pages/configure-root-password-page.ts

```diff
@@ -0,0 +1,22 @@
+export class ConfigureRootPasswordPage {
+    readonly page: Page;
+    readonly password: Locator;
+    readonly confirmPassword: Locator;
+    readonly confirmButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.password = page.getByRole('textbox', { name: 'User password', exact: true });
+        this.confirmPassword = page.getByLabel('Password confirmation');
+        this.confirmButton = page.getByRole('button', { name: 'Confirm' });
+    }
+
+    async fillAndConfirmPassword(password: string) {
```

Comment:
actually keeps being confusing, because user of this function would think that it clicks on confirm, it is short of `fillPasswordAndConfirmPassword` and there is nothing better if you want to group functionality....but I regret my own suggestion, feel free to not change this now :)